orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/resource/EmrResource.scala

```diff
@@ -26,6 +26,7 @@ case class EmrResource(
   loggingPath: String,
   spec: EmrResource.Spec,
   lastAttempt: Boolean,
+  spotInstance: Boolean,
```

Comment:
do we need this one? When `InstanceGroupConfig.instanceBidPrice` is `None`, it would be on demand, no?